lms/templates/certificates/_certificate-supplemental.html

```diff
@@ -0,0 +1,36 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
NOTE: @marcotuts and @mattdrayer, I've kept this semi edX-specific content in the codebase, but abstracted it out into this include template. Its not being called, but I'd like a call on whether or not this type of content is MVP for what we need.